app/src/main/kotlin/com/wire/android/ui/common/UserBadge.kt

```diff
@@ -28,12 +28,16 @@ import com.wire.kalium.logic.data.user.ConnectionState
 fun UserBadge(
     membership: Membership,
     connectionState: ConnectionState? = null,
+    isDeleted: Boolean = false,
     startPadding: Dp = dimensions().spacing0x,
     topPadding: Dp = dimensions().spacing0x
 ) {
     if (connectionState == ConnectionState.BLOCKED) {
         Spacer(modifier = Modifier.width(startPadding))
         BlockedLabel()
+    } else if (isDeleted) {
+        Spacer(modifier = Modifier.width(startPadding))
+        DeletedLabel()
```

Comment:
I don't know which is better but I see that Spacer should not be added when none of these ifs are true 🤔

Reply:
As a rule of thumn I'd use the `when` clause the moment there is a second  `if` on the same block. Regarding the extra spacer, I think it would still not harm having some extra padding at the begginning of the badge, but also feel free to add it only if any the conditions match.